observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-configuring.adoc

```diff
@@ -37,4 +37,9 @@ The {TempoOperator} supports monitoring and alerting of each TempoStack componen
 
 include::modules/distr-tracing-tempo-configuring-tempostack-metrics-and-alerts.adoc[leveloffset=+2]
 
+[role="_additional-resources"]
+[id="additional-resources_distr-tracing-tempo-configuring-tempostack-metrics-and-alerts"]
+==== Additional resources
+* xref:../../../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[Enabling monitoring for user-defined projects]
+
```

Comment:
Both of these are deep enough in levels that they don't appear in the TOC, so I wonder if they could isntead be consolidated as a single `==` "Additional resources" at the bottom of the overall assembly?

Reply:
@adellape, thank you for the suggestion. When I was adding these `Additional resources`, my primary concern was their fundability. As a reader, when I finish reading a section, I already see the section's additional resources on the next lines. Whereas, if I put them at the bottom of the page, it will be harder for readers to mentally correlate each additional resource to a section because the additional resource is about another topic; also, it might not occur to some readers to check at the bottom of the page. It is already unfortunate that xrefs can't be used in modules bcause this in turn means that xrefs can't be used inline, which is a pity.

Reply:
Fair enough!